### DISCUSSION TITLE
How confident is the algorithm that I am at least as good as mu - sigma?

### DISCUSSION THREAD

**Discussion by jauggy** [OpenDebates/openskill.py#133] [Q&A] [answered]
Say the algorithm knows my mu and sigma. How confident is the library that I am at least as good as mu - sigma?
And for mu-3*sigma is the algorithm close to 100% confident I am at least as good as that number?

**Comment by vivekjoshy** [comment, marked as answer]
$$\Pr(\mu-1\sigma \le X \le \mu+1\sigma) \approx 68.27\%$$

$$\Pr(\mu-2\sigma \le X \le \mu+2\sigma) \approx 95.45\%$$

$$\Pr(\mu-3\sigma \le X \le \mu+3\sigma) \approx 99.73\%$$

[Source](https://en.wikipedia.org/wiki/68%E2%80%9395%E2%80%9399.7_rule)
